assets/markdown/remapURLs.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function (value) {
+  var userRegex = /(<a href=")(\/[u]\/[a-zA-Z_\/-]+)(">)(.*)(<\/a>)/g;
+  var usersReplaced = value.replace(userRegex, "$1https://www.reddit.com$2$3$4$5 ($1$2$3FlairHQ$5)");
+  var subRegex = /(<a href=")(\/[r]\/[a-zA-Z_\/-]+)(">)(.*)(<\/a>)/g;
```

Comment:
We should change the `.*` here to `.*?`, otherwise it'll do a greedy search and might map more than we want it to.

Also, note that it won't correctly link `u/username` or `r/subreddit` like reddit does.